src/coldfront_plugin_api/scim_v2/filters.py

```diff
@@ -0,0 +1,19 @@
+'''
+Defines the SCIM filters for coldfront users and groups
+'''
+
+from django_scim.filters import UserFilterQuery, GroupFilterQuery
+from django_scim import constants
+from django_scim.utils import get_user_model
+
+class ColdfrontUserFilterQuery(UserFilterQuery):
```

Comment:
Can you briefly explain what this and `attr_map` does?

Reply:
`UserFilterQuery` is the class used by `django-scim` to translate a SCIM filter query (i.e `filter=uesrName eq foo`) into a raw SQL query. 

For the translation to work, the query class needs to know which SCIM property maps to which SQL table column. This mapping must be defined with a `attr_map` dict (i.e I've mapped `emails.value` to `email`, so every SCIM filter query asking for `emails.value` will query the `email` field in the SQL database).

The internals of `UserFilterQuery` is related to the `scim2-filter-parser` module, made by the same organization. I mostly juts followed their [example](https://github.com/15five/scim2-filter-parser?tab=readme-ov-file#use)

Reply:
Awesome, thanks for the explanation.

Please provide that as a class docstring, including the link too would be helpful. I'm generally not very fond of module level dosctrings.

Also, please use `"""` instead of `'''` for docstrings and be mindful of various other Python styling guidelines.